Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -12,15 +12,18 @@ This guide assumes a Rook cluster as explained in the [Quickstart](../../Getting
 
 Rook can configure the Ceph Object Store for several different scenarios. See each linked section for the configuration details.
 
-1. Create a [local object store](#create-a-local-object-store) with dedicated Ceph pools. This option is recommended if a single object store is required, and is the simplest to get started.
+1. Create a [local object store](#create-a-local-object-store-with-s3) with dedicated Ceph pools. This option is recommended if a single object store is required, and is the simplest to get started.
 2. Create [one or more object stores with shared Ceph pools](#create-local-object-stores-with-shared-pools). This option is recommended when multiple object stores are required.
 3. Connect to an [RGW service in an external Ceph cluster](#connect-to-an-external-object-store), rather than create a local object store.
 4. Configure [RGW Multisite](#object-multisite) to synchronize buckets between object stores in different clusters.
 
 !!! note
     Updating the configuration of an object store between these types is not supported.
+Rook has the ability to either deploy an object store in Kubernetes or to connect to an external RGW service.
```

Comment:
There needs to be an empty line before the new paragraph, or this won't render online correctly.